micro_epsilon_scancontrol_driver/src/driver.cpp

```diff
@@ -5,35 +5,33 @@ namespace scancontrol_driver
 
     static const rclcpp::Logger LOGGER = rclcpp::get_logger("scancontrol_driver");
 
-    ScanControlDriver::ScanControlDriver(rclcpp::Node::SharedPtr& nh, rclcpp::Node::SharedPtr& private_nh)
+    ScanControlDriver::ScanControlDriver():Node("scancontrol_driver")
```

Comment:
Clang warns about other things like having a parenthesis on these return codes:
```
if (return_code = ScanControlDriver::StartProfileTransfer() < GENERAL_FUNCTION_OK){
```

But the initializer list is not a problem. I'll change the return code issues in a separate PR because that's not related to the ROS2 fix